app/lib/cfe_connection.rb

```diff
@@ -25,11 +25,12 @@ def create_proceeding_type(assessment_id, proceeding_type)
     create_record(assessment_id, "proceeding_types", proceeding_types:)
   end
 
-  def create_applicant(assessment_id, date_of_birth:, receives_qualifying_benefit:)
+  def create_applicant(assessment_id, date_of_birth:, receives_qualifying_benefit:, employed:)
     applicant = {
```

Comment:
Over the past fortnight we've very slowly been moving logic out of CfeConnection into the Submit services that call it. `create_applicant` is one of the very few to be left. As part of this ticket, any chance you could move the creation of the applicant hash into SubmitApplicantService, and then pass it as a param to `create_applicant`? That would help us on our slow quest towards consistency.